users_db.py

```diff
@@ -0,0 +1,48 @@
+"""Module describes base class for interacting with sqlite3 database and set of specific queries for user database"""
+import sqlite3
+
+
+class DatabaseManager:
+    """This class provides connection to database and sending queries"""
+    DB_PATH = "default_database.db"
+
+    def __init__(self, db_path=None):
+        """Intializes path to database and creates connection and cursor objects"""
+        if db_path is not None:
```

Comment:
make it easier:
def __init__(self, db_path="default_database.db"):
self.conn = sqlite3.connect(db_path, check_same_thread=False)

Reply:
fixed